backend/grant/user/models.py

```diff
@@ -246,6 +246,31 @@ def set_silenced(self, is_silence: bool):
         db.session.add(self)
         db.session.flush()
 
+    def set_admin(self, is_admin: bool):
+        # TODO: audit entry & possibly email user
```

Comment:
I actually like the added security-through-obscurity of not emailing the user. This means that if an admin somehow mistakenly admins someone, they can undo it with little harm.